libbeat/metric/system/host/host.go

```diff
@@ -68,7 +68,7 @@ func ReportInfo(_ monitoring.Mode, V monitoring.Visitor) {
 	}
 	info := h.Info()
 
-	monitoring.ReportString(V, "name", info.Hostname)
+	monitoring.ReportString(V, "hostname", info.Hostname)
```

Comment:
@ycombinator This affects the monitoring reporting. Will this break something?

Reply:
I checked docs in the `.monitoring-beats-*` indices, and this field will end up as `beats_state.state.host.hostname` in the monitoring docs after this PR, in place of `beats_state.state.host.name` today. The latter is not being used anywhere in the Stack Monitoring UI code, so this change should be safe.

We will however, want to update the mapping here: https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/core/src/main/resources/monitoring-beats.json#L52-L54.